Content.Client/_EE/Flight/Components/FlightVisualsComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Robust.Client.Graphics;
+using Robust.Shared.GameStates;
+
+namespace Content.Client._EE.Flight.Components;
```

Comment:
Unused using directive
```suggestion
using Robust.Client.Graphics;

namespace Content.Client._EE.Flight.Components;
```